src/index.html

```diff
@@ -8,6 +8,7 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link rel="stylesheet" href="../node_modules/xel/stylesheets/material.theme.css">
   <script src="../node_modules/xel/xel.min.js"></script>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
might want to consider downloading this and adding it to our project so we're not dependent on stuff from the web